src/transformers/testing_utils.py

```diff
@@ -2083,7 +2083,7 @@ def nested_simplify(obj, decimals=3):
         return nested_simplify(obj.numpy().tolist())
     elif isinstance(obj, float):
         return round(obj, decimals)
-    elif isinstance(obj, (np.int32, np.float32)):
+    elif isinstance(obj, (np.int32, np.float32, np.float16)):
```

Comment:
Why do we need to change that ? The fp16 support was working in the past for pipeline. Is it because you are using it in your tests ?

Reply:
Yes, it is for the fp16 tests.

Reply:
sounds good !